### PR TITLE
fix: duplicate iframes in cal video

### LINKED ISSUE #6658: [CAL-882] `/video`: cal video renders duplicate iframes
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-882](https://linear.app/calcom/issue/CAL-882/video-cal-video-renders-duplicate-iframes)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #6658

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/pages/video/[uid].tsx [hunk 1]
```diff
@@ -36,19 +39,20 @@ export default function JoinCall(props: JoinCallPageProps) {
...
-        : null),
-    });
-  }, [props.booking?.references]);
+    callFrame.join();
+    return () => {
+      callFrame.destroy();
+    };
+  }, []);
+
+  const title = `${APP_NAME} Video`;
```
Comment (G3root) [inline apps/web/pages/video/[uid].tsx:51]: this change fixed this warning.

`
Warning: A title element received an array with more than 1 element as children. In browsers title Elements can only have Text Nodes as children. If the children being rendered output more than a single text node in aggregate the browser will display markup and comments as text in the title and hydration will likely fail and fall back to client rendering
`
for more context : https://github.com/vercel/next.js/discussions/38256

Comment (emrysal) [reply apps/web/pages/video/[uid].tsx:51]: NIT I'm a proponent here for inline substitution, as it is unlikely the `title` var will be used anywhere else and is harder to read over inline.

```jsx
<title>{`${APP_NAME} Video`}</title>
```

File: apps/web/pages/video/[uid].tsx [hunk 2]
```diff
@@ -36,19 +39,20 @@ export default function JoinCall(props: JoinCallPageProps) {
         width: "100%",
         height: "100%",
       },
+      url: meetingUrl,
```
Comment (emrysal) [inline apps/web/pages/video/[uid].tsx:42]: NIT Should be parsed with Zod and then destructured into here.

**Review by emrysal — APPROVED** [review]
Excellent improvements LGTM

**Pull request merged**